go.mod

```diff
@@ -3,23 +3,24 @@ module github.com/lightninglabs/pool
 go 1.16
 
 require (
-	github.com/btcsuite/btcd v0.22.0-beta.0.20220207191057-4dc4ff7963b4
-	github.com/btcsuite/btcd/btcec/v2 v2.1.0
-	github.com/btcsuite/btcd/btcutil v1.1.0
+	github.com/btcsuite/btcd v0.22.0-beta.0.20220316175102-8d5c75c28923
+	github.com/btcsuite/btcd/btcec/v2 v2.1.3
+	github.com/btcsuite/btcd/btcutil v1.1.1
+	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/btcwallet v0.14.0
+	github.com/btcsuite/btcwallet v0.14.1-0.20220322182735-b0001c262734
 	github.com/btcsuite/btcwallet/wallet/txrules v1.2.0
 	github.com/btcsuite/btcwallet/wtxmgr v1.5.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1
 	github.com/golang/mock v1.6.0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/aperture v0.1.17-beta.0.20220325093943-42b9d4c1be7f
-	github.com/lightninglabs/lndclient v0.15.0-0
+	github.com/lightninglabs/aperture v0.1.17-beta.0.20220328072456-4a2632d0be38
+	github.com/lightninglabs/lndclient v1.0.1-0.20220328143152-e7bebec99366
```

Comment:
This has a tag now: `v0.15.0-1`

Reply:
I tried it before. I am not sure if I screwed up with the git tags, but if I use the `v0.15.0-1` I get

```
➜  pool git:(signoutputraw) go mod tidy      
go: finding module for package github.com/btcsuite/btcd/btcec
github.com/lightninglabs/pool imports
	github.com/lightninglabs/lndclient imports
	github.com/btcsuite/btcd/btcec: module github.com/btcsuite/btcd@latest found (v0.18.1), but does not contain package github.com/btcsuite/btcd/btcec
```

Reply:
Okay, there was something very weird with that tag... Created a new one (`v0.15.0-2`) and it seems to work now.